.github/workflows/tests.yml

```diff
@@ -0,0 +1,179 @@
+name: tests
+on:
+  pull_request:
+    branches:
+      - main
+    
+env:
+  ARM_CLIENT_ID: 6c2c6744-1e33-4ee0-b50a-2fdce243491e
+  ARM_CLIENT_SECRET: ${{ secrets.AZ_CLIENTSECRET_EXP_JUN2024 }}
+  ARM_ENVIRONMENT: public
+  ARM_LOCATION: westus
+  ARM_SUBSCRIPTION_ID: 0282681f-7a9e-424b-80b2-96babd57a8a1
+  ARM_TENANT_ID: 706143bc-e1d4-4593-aee2-c9dc60ab9be7
+  PROVIDER: "pulumi-azure-justrun"
+  # THIS GITHUB_TOKEN IS A REQUIREMENT TO BE ABLE TO WRITE TO GH RELEASES
```

Comment:
I think we can delete all of these comments as they're instructions from the boilerplate